### ISSUE TITLE
Filter by custom dates in Test Results page.

### ISSUE THREAD

**Issue by nditada** [instedd/cdx#939]
We need to add a date range selector.

@nekron can you link to the design widget for this?

**Comment by nekron** [comment]
https://projects.invisionapp.com/d/main#/console/4103589/117713806/preview

**Comment by pmallol** [comment]
Currently the date range picker is only implemented in the Home page. Other Date filters, such as the one in Tests, just have the 'Previous Month/Week/year' options.

@nekron should the other Date filters have a 'date range' option as well?

**Comment by nekron** [comment]
Yes, as #840 partially says, all date inputs **and filters** should be controlled by calendar tools.

**Comment by pmallol** [comment]
As for Version 0.13-pre1 date range picker still missing from Test page.
